File: Java/Math/Count Squares.java
//https://www.geeksforgeeks.org/problems/count-squares3649/1
class Solution {
    static int countSquares(int n) {
        // code here
        int sum = 0;
    for (int i = 1; i <= Math.sqrt(n); i++) {
        if (i * i < n) {
            sum++;
        } else {
            break;
        }
    }
    return sum;
    }
}

File: Python/Math/Count Squares.py
//https://www.geeksforgeeks.org/problems/count-squares3649/1
class Solution:
    def countSquares(self, n):
        # code here 
        sum = 0
        for i in range(1, int(math.sqrt(n)) + 1):
            if i * i < n:
                sum += 1
            else:
                break
        return sum
